### PR TITLE
Sync various sections

### PR DESCRIPTION
**Pull request by saundefined**
Apply
- php/doc-en@62aa2694baf7fe904744a9b0f8a2a26e7fc3cf12 (remaining)
- several `reference/zip/constants.xml` commits

### CODE REVIEW

**Comment by saundefined** [comment]
Contains #524 changes

File: reference/zip/constants.xml [hunk 1]
```diff
@@ -123,6 +128,10 @@
     </simpara>
    </listitem>
   </varlistentry>
+ </variablelist>
+
+ <variablelist xml:id="ziparchive.constants.fl">
+  <title>Archiver les options</title>
```
Comment (Girgias) [inline reference/zip/constants.xml:134]:
```suggestion
  <title>Les options d'archivage</title>
```

File: reference/zip/constants.xml [hunk 2]
```diff
@@ -74,6 +75,10 @@
     </simpara>
    </listitem>
   </varlistentry>
+ </variablelist>
+
+ <variablelist xml:id="ziparchive.constants.afl">
+  <title>Archiver les drapeaux globaux</title>
```
Comment (Girgias) [inline reference/zip/constants.xml:81]:
```suggestion
  <title>Les drapeaux globaux d'archivage</title>
```

File: reference/zip/constants.xml [hunk 3]
```diff
@@ -9,14 +9,15 @@
...
+   Il existe différents types de constantes, dont les principaux sont les suivants :
    Les drapeaux globaux (préfixés par <literal>AFL_</literal>),
    les options (préfixées par <literal>FL_</literal>),
    les erreurs (préfixées par <literal>ER_</literal>)
    ou les modes (sans préfixe).
   </para>
 
- <variablelist>
+ <variablelist xml:id="ziparchive.constants.mode">
+  <title>Modes de l'archive</title>
```
Comment (Girgias) [inline reference/zip/constants.xml:20]:
```suggestion
  <title>Modes d'ouverture de l'archive</title>
```

**Review by Girgias — APPROVED** [review]

**Pull request merged**